velox/expression/tests/ArrayProxyTest.cpp

```diff
@@ -278,5 +278,43 @@ TEST_F(ArrayProxyTest, testTimeStamp) {
   assertEqualVectors(result, makeNullableArrayVector(expected));
 }
 
+TEST_F(ArrayProxyTest, testString) {
+  auto result =
+      prepareResult(std::make_shared<ArrayType>(ArrayType(VARCHAR())));
+
+  exec::VectorWriter<ArrayProxyT<Varchar>> writer;
+  writer.init(*result.get()->as<ArrayVector>());
```

Comment:
nit: the `.get()` is redundant, and consider `asUnchecked` to avoid the `dynamic_cast`.

`*result->asUnchecked<ArrayVector>()`